Mathlib/Tactic/Linter/DocString.lean

```diff
@@ -24,6 +24,25 @@ register_option linter.style.docString : Bool := {
   descr := "enable the style.docString linter"
 }
 
+/--
+Extract all `declModifiers` from the input syntax. We later extract the `docstring` from it,
+but we avoid extracting directly the `docComment` node, to skip `#adaptation_note`s.
+-/
+def getDeclModifiers : Syntax → Array Syntax
+  | s@(.node _ kind args) =>
+    (if kind == ``Parser.Command.declModifiers then #[s] else #[]) ++ args.flatMap getDeclModifiers
+  | _ => #[]
+
+/--
+Currently, this function simply removes `startColumn` spaces after each `\n`
+in the input string `docString`.
+
+If/when the `docString` linter expands, it may take on more string processing.
+-/
+def deindentString (startColumn : Nat) (docString : String) : String :=
```

Comment:
What do you think about calling `startColumn` `dedent` instead? (Not a strong opinion, perhaps your name is better.)

Reply:
I went for `currIndent`, which feels like it takes the good of both options!